### PR TITLE
Initial 

### PR DESCRIPTION
**Pull request by zacblazic**
The initial creation of the `fsfreeze` docker image.

Credits to [velero](https://github.com/heptio/velero/blob/9e7ff4e3d9f2a3ab17f645f28b05931527fe94e6/Dockerfile-fsfreeze-pause) for the original contents of the dockerfile.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          version: 18.06.0-ce
+      - run:
+          name: "Build Image"
+          command: docker build -t $IMAGE_NAME .
+      - run:
+          name: "Test Image"
+          command: docker run $IMAGE_NAME fsfreeze --version
+
+workflows:
+  version: 2
```
Comment (hadrianvalentine) [inline .circleci/config.yml]: According to CircleCI, workflow number isn't actually a thing in config version `2.1` anymore
Screenshot 2019-06-10 at 18 13 45
https://discuss.circleci.com/t/circleci-2-1-config-overview/26057

Comment (zacblazic) [reply .circleci/config.yml]: Addressed in https://github.com/Intellection/docker-fsfreeze/pull/1/commits/5e5df4ad8cd1e367280f308f59b46f3169d36ee6.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+The [`fsfreeze`][1] command is used to suspend and resume access to a file system. This allows consistent snapshots to be taken by halting any new write system calls while allowing ongoing transactions to complete.
+
+Supported filesystems:
+* Ext3
+* Ext4
+* ReiserFS
+* JFS
+* XFS
+
+Note: `fsfreeze` is unncessary for device-mapper devices. The device-mapper (and LVM) automatically freezes filesystem on the device when a snapshot creation is requested.
```
Comment (hadrianvalentine) [inline README.md]: *unnecessary

Comment (zacblazic) [reply README.md]: Hah, I copied it from the [man page](https://linux.die.net/man/8/fsfreeze).

Comment (zacblazic) [reply README.md]: Addressed in https://github.com/Intellection/docker-fsfreeze/pull/1/commits/0572d7cff2db4dd269feb835ff4bec47753e669f.

**Review by hadrianvalentine — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (hadrianvalentine) [reply README.md]: Haha
Whoops

**Comment by itskingori** [comment]
👍

**Pull request merged**